resources/lang/deu.ini

```diff
@@ -0,0 +1,100 @@
+; NOTE: We use the word newspaper for the following two meanings: the newspaper office and the newspaper (reading).
+
+; This string is used with ~.perm.* strings
+main.perm.base=Du hast nicht die Berechtigung für {%0}
+main.perm.generic=diese Aktion ausführen
+main.error.sub.invNoSpace=Ein paar Zeitungen aus der Warteschlange {%0} konnten nicht zu deinem Inventar hinzugefügt werden, da nicht genug Platz vorhanden ist. Sie werden sich weiterhin in der Warteschlange befinden, bis genügend Platz frei ist.
+command.onlyPlayers=Nur Spieler können diesen Befehl ausführen
+gui.buy.info.noFee=Dieser Server scheint kein Wirtschaftssystem zu verwenden, deswegen wird kein Preis berechnet.
+
+gui.buyinfo.title=Information für {%0}
+gui.buyinfo.label.desc=Beschreibung: {%0}
+gui.buyinfo.label.member=Mitglied(er): {%0}
+gui.buyinfo.label.priceOne=Einzelpreis: {%0}
+gui.buyinfo.label.priceSub=Abo-Gebühr: {%0}
+gui.buyinfo.button.buy=Kaufen für {%0}
+gui.buyinfo.button.subscribe=Abonnieren für {%0}
+gui.buyinfo.perm.subscribe=Zeitungen abonnieren
+gui.buyinfo.error.alreadySubscribe=Du hast diese Zeitung bereits abonniert.
+gui.buyinfo.error.noMoney="Du hast nicht genug Geld, um diese Zeitung zu abonnieren. Du brauchst {%0} mehr."
```

Comment:
The quotation mark isn't needed for this since your translation doesn't use semicolon in it. Also, the original string doesn't have period at the end. Can you remove it from your translation? Thanks!